flink-ai-flow/QUICKSTART.md

```diff
@@ -2,117 +2,108 @@
 
 ```shell
 cd flink-ai-extended
-bash flink-ai-flow/bin/install_aiflow.sh
+sh flink-ai-flow/bin/install_aiflow.sh
 ```
-To make it easier to start server, some shell scripts like start-aiflow.sh will be installed with upon commands.
+Some shell scripts like start-aiflow.sh will be installed with upon commands.
 You could run with `which start-aiflow.sh` to check if the scripts are installed successfully. 
-If not, you can reinstall with `sudo` command. Any other problems during the installation, please refer to the [Troubleshooting](#troubleshooting) section to see if it can help.
-If you meet any problems during the installation, please refer to the [Troubleshooting](#troubleshooting) section to see if it can help.
+Depends on your OS, if permission error occurred, you can rerun the install script with `sudo` command. 
 
-## Start AI Flow
+Any other problems during the installation, please refer to the [Troubleshooting](#troubleshooting) section to see if it can help.
 
 ### Start Servers
-Run following command to start Notification service, AI Flow Server and Airflow Server:
-
-```shell
-start-aiflow.sh
-```
+As you may know AI Flow contains three long running services, Notification service, AI Flow Server and Scheduler service(Apache Airflow by default), 
+you could start all services with a single scripts. 
 
-If you execute this command for the first time, you will get the following output:
+The mysql database that you created during [Prerequisites](#prerequisites) should be passed as parameter to store metadata of AI Flow
 
-```text
-The ${AIRFLOW_HOME}/airflow.cfg is not exists. You need to provide a mysql database to initialize the airflow, e.g.:
-start-aiflow.sh mysql://root:root@127.0.0.1/airflow
+```shell
+start-aiflow.sh mysql://user:password@ip_or_hostname/airflow
 ```
 
-Please prepare the MySQL database refer to [Prerequisites](#prerequisites-1) and rerun the `start-aiflow.sh` with the MySQL parameter.
 After that you will get the output like:
 
 ```text
-Scheduler log: ${AIRFLOW_HOME}/scheduler.log
-Scheduler pid: 69945
-Web Server log: ${AIRFLOW_HOME}/web.log
-Web Server pid: 69946
-Master Server log:  ${AIRFLOW_HOME}/master_server.log
-Master Server pid: 69947
+Admin user admin created
+Notification service log: ${AIRFLOW_HOME}/logs/notification_service.log
+Notification service pid: 21859
+Scheduler log: ${AIRFLOW_HOME}/logs/scheduler.log
+Scheduler pid: 21860
+Web Server log: ${AIRFLOW_HOME}/logs/web.log
+Web Server pid: 21861
+Master Server log: ${AIRFLOW_HOME}/logs/master_server.log
+Master Server pid: 21862
 Airflow deploy path: ${AIRFLOW_HOME}/airflow_deploy
 Visit http://127.0.0.1:8080/ to access the airflow web server.
 ```
 If you see the login page when visit http://127.0.0.1:8080/, you can login with the defaut user name(admin) and password(admin).
+This is the scheduler web UI which you can see the workflows that you summitted.
 
-### Prepare AI Flow Project
```

Comment:
It's recommended to introduce the structure of the project.

Reply:
IMO it should be introduced in tutorial doc by @bgeng777